davidpak/SmartNote
# Conda env

## What type of PR is this? (check all applicable)


## Description
Created an anaconda environment that contains all necessary dependencies for the project.


## QA Instructions, Screenshots, Recordings
To create the environment, run:

`conda env create -f environment.yml`

To activate the environment run:

`conda activate smartnote_env`

To exit out of the environment run:

`conda deactivate smartnote_env`

To ensure that all dependencies are correctly installed run:

`pip list`

while in the conda environment.

_For more info, check out the
[Forem Accessibility Docs](https://developers.forem.com/frontend/accessibility)._

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._

      have not been included_
